### ISSUE TITLE
Investigate and fix build error in `wasm-test-runner`

### ISSUE THREAD

**Issue by th4s** [tlsnotary/tlsn#564]
Currently our `wasm-test-runner` fails with 
```
error[E0599]: no method named `clone` found for struct `JsStatic` in the current scope
  --> /home/th4s/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wasm-bindgen-rayon-1.2.1/src/lib.rs:81:13
   |
81 |         URL.clone()
   |             ^^^^^ method not found in `JsStatic<JsString>`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `wasm-bindgen-rayon` (lib) due to 1 previous error
```
To reproduce go into `wasm-test-runner` and do a `cargo clean && cargo update` then execute `./run.sh`.

**Comment by th4s** [comment]
Probably related to a recent update of `js-sys` but I could not get it working spending a few minutes on downgrading `js-sys`. Probably more crates need to be down-graded.

**Comment by th4s** [comment]
Eventually related: https://github.com/rustwasm/wasm-bindgen/pull/4057

**Comment by th4s** [comment]
Closed by #567
